### ISSUE TITLE
database location

### ISSUE THREAD

**Issue by jiarong** [thackl/pro-tycheposons#2]
Hi, are the databases needed for  for `element-finder/bin/irve_finder` all in `element-finder/data` except for virus HMMs from virsorter? 
For virus HMMs, which version of virsorter did you use? and HMMs include virome (Phage_gene_catalog_plus_viromes) or not (Phage_gene_catalog)?

thanks!

**Comment by thackl** [comment]
Hi @jiarong,

sorry for the delay, I am currently on holidays. Good question. @iimog, do you know/have a backup of your notes that can tell us what files you used to create virsorter_Pool_clusters.hmm? Unfortunately, your files have been deleted from the MIT servers some time after you left, so I cannot investigate myself.

**Comment by jiarong** [comment]
@thackl, if you can find the size of `virsorter_Pool_clusters.hmm`, I should be able to find out which version of virsorter and whether it includes virome or not.

**Comment by iimog** [comment]
Hi @jiarong, same excuse here: I just came back from holidays :smiley: I checked my local files and found a copy of the `virsorter_Pool_clusters.hmm` file. It is about 900MB in size (937430800 bytes to be exact) and the md5sum is `6f26575119bde2a2ba25df1c5e0e1b92`. If you tell me how to find out the Version of `virsorter`, I can do it. I can also send the hmm file to you. The `DATE` field in the hmm file is all `Wed Mar 26 <TIME> 2014`. Hope that helps!
